Chat/AppManager.h

```diff
@@ -26,9 +26,13 @@ class AppManager
     void GetMsgLoop(); //message getting
 
     void SendMsg(std::string msg); //after user wrote message in chat
-    void ActivateCommand(std::string msg); //called by buttons with different commands
+    void PutMsg(); //will be used by recv loop
+
+    void* ActivateCommand(std::string msg); //called by buttons with different commands
                                      //example: ChangeMsgColor, ChangeName, ChangeHomeDir, DownloadSomeone`sFile
```

Comment:
well, in theory it is name color, but i don't know if there is an option to make part of the string as special color